packages/cli/src/cli/cogify/action.cog.ts

```diff
@@ -208,10 +207,6 @@ export class CommandCogCreate extends CommandLineAction {
 
     const cfg = getDefaultConfig();
     if (expectedTiffs.size === 0) {
-      // Insert Imagery and TileSet Config
-      await insertConfigImagery(cfg, job, logger);
-      await insertConfigTileSet(cfg, job, logger);
-
       // Update job status if this is the processing job.
```

Comment:
all of the processingId stuff needs to go too.

Reply:
Yes, I will make another pr to remove all the processId.